tests/unit/s2n_psk_test.c

```diff
@@ -434,7 +434,7 @@ int main(int argc, char **argv)
             "ff5dd36d64e861be7fd61d2827db279cce145077d454a3664d4e6da4d29ee03725"
             "a6a4dafcd0fc67d2aea70529513e3da2677fa5906c5b3f7d8f92f228bda40dda72"
             "1470f9fbf297b5aea617646fac5c03272e970727c621a79141ef5f7de6505e5bfb"
-            "c388e93343694093934ae4d357fad6aacb");
+            "c388e93343694093934ae4d357");
         S2N_BLOB_FROM_HEX(client_hello_prefix,
```

Comment:
What's with this change? Why was it working before if the value was wrong?

Reply:
Ref: https://github.com/aws/s2n-tls/pull/2722#discussion_r614254371